v_frame/src/plane.rs

```diff
@@ -428,11 +429,26 @@ impl<T: Pixel> Plane<T> {
     &mut self, source: &[u8], source_stride: usize, source_bytewidth: usize,
   ) {
     let stride = self.cfg.stride;
-    for (self_row, source_row) in self
-      .data_origin_mut()
-      .chunks_mut(stride)
```

Comment:
how far is using `chunks_exact`  from this optimality?

Reply:
`chunks_exact` uses a couple of integer division instructions at the start of the loop to get the amount of chunks, while the manual indexing does not require these operations. However, it seems like the approach with `step_by` has more instructions to carry out the inner loop, see here: https://godbolt.org/z/MTc8PG96W

I'll see if I can find something that avoids integer division while also having a small inner loop

Reply:
I decided to just use chunks_exact